### PR TITLE
[Tests-Only] Adjust pact interactions for provider tests for signedUrlIntegration

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
In this PR, the signedUrlIntegration test file is adjusted such that the tests pass on the provider side as well.

## Related Issue: 
- https://github.com/owncloud/owncloud-sdk/issues/709

### CODE REVIEW

File: tests/webdavHelper.js [hunk 1]
```diff
@@ -82,8 +82,18 @@ const deleteItem = function (user, password, itemName) {
   })
 }
 
+const getSignKey = function (endpoint, username, password) {
```
Comment (dpakach) [inline tests/webdavHelper.js:85]:
```suggestion
const getSignKey = function (username, password) {
```
endpoint is always same

Comment (jasson99) [reply tests/webdavHelper.js]: fixed

**Review by dpakach — COMMENTED** [review]
Lgtm, I dont have much idea about how signed urls work, @individual-it maybe you can review this

**Review by individual-it — CHANGES_REQUESTED** [review]
good job, but downloading a signed url seems not to work yet
image

**Review by individual-it — APPROVED** [review]

**Pull request merged**